app/views/dashboards/show.html.erb

```diff
@@ -64,8 +64,8 @@
                                           flags: "progressive",
                                           fetch_format: "auto",
                                           sign_url: true),
-                                          class: "sloan mb-7",
-                                          alt: "#{SiteConfig.mascot_image_description}") %>
+                            class: "sloan mb-7",
+                            alt: SiteConfig.mascot_image_description.to_s) %>
```

Comment:
this is because the two attributes belong to the outer function, `image_tag`, not `cl_image_path`